### PR TITLE
Updated branch description

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
As we are no longer using a dev branch, there should be no reference of it in our readme

### CODE REVIEW

**Comment by eHammarstrom** [comment]
Note that this information is also located in the `CONTRIBUTIONS.md` and is to be removed from the `README.md` at a later date. There is no need for duplicated branch information. We're also still discussing the branch structure and it is to be determined.

**Comment by ErikBjare** [comment]
@Mattemagikern Why was this closed? I think it should be reopened and have the following actions taken:

 - [x] Remove branching info from CONTRIBUTIONS as well. 
 - [ ] Create a link to CONTRIBUTIONS from the README. 

Then, once branch structure is determined (will likely take a while since things won't be production ready anytime soon) we can add back info about it to the CONTRIBUTIONS document.

**Comment by axelwihlborg** [comment]
Updated the documents to what Erik said somewhat.
We should keep branch info in one file and one file only to avoid Double Maintenance problem.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Website
-This is the (soon to be) official website of the computer science guild, commit to make a difference! 
+[![Build Status](https://travis-ci.org/Dsek-LTH/Website.svg?branch=master)](https://travis-ci.org/Dsek-LTH/Website)
```
Comment (ErikBjare) [inline README.md:2]: Nice

File: README.md [hunk 2]
```diff
@@ -12,10 +18,17 @@ This is the (soon to be) official website of the computer science guild, commit
 * Write great commit messages and pull requests.
 This includes to use standard features like "closing #*".
 
+# The next big step
+If you are taking the next step towards writing code and wants to know how its done, read [Contributions](https://github.com/Dsek-LTH/Website/blob/master/CONTRIBUTIONS.md). If you can't find what you are looking for, contact either one of the Admins.
+If you questions is of a bigger nature, such as collaboration with Dsek, our development team or questions regarding the project as whole, please contact [Mattemagikern](https://github.com/Mattemagikern) or [Axel Wihlborg](https://github.com/axelwihlborg).
+
 ## Branches
-Currently we have one main branch and in a later stage we will have a production branch with the current state of the webpage. The production branch will be unreachable for everyone except the Admin team and they are responsible for merging major, stable changes to it.
+Currently we have 2 main branches, master and dev. dev is the development branch, all pull requests shall be aimed at dev.
```
Comment (ErikBjare) [inline README.md]: The problem is that we don't have two branches right now (nor a need for two branches). I'd say this should be removed entirely until a proper decision is made that there should be more than one branch.

**Comment by eHammarstrom** [comment]
Removed `dev` from Branches section. Suggested `feature` branches which may be useful for major changes running in "parallel" with the master branch.

**Comment by ErikBjare** [comment]
Looks great to me!

**Review by ErikBjare — APPROVED** [review]

**Pull request merged**